build.sbt

```diff
@@ -135,7 +135,7 @@ lazy val connector = (project in file("connector"))
 
       // Keep com.google.cloud dependencies in sync
       "com.google.cloud" % "google-cloud-bigquery" % "1.131.1",
```

Comment:
Shouldn't google-cloud-bigquery be upgraded to 2.1.7 or is it an unrelated change?

Reply:
google-cloud-bigquerystorage update to 2.1.2 is only necessary for handling some new RESOURCE_EXHAUSTED errors that may be sent from the bq storage read api calls. I don't know why we would need to update google-cloud-bigquery for that reason.